helm/flink-kubernetes-operator/templates/flink-operator.yaml

```diff
@@ -115,7 +119,7 @@ spec:
             - name: FLINK_PLUGINS_DIR
               value: /opt/flink/plugins
             - name: LOG_CONFIG
-              value: -Dlog4j.configurationFile=/opt/flink/conf/log4j-operator.properties
+              value: {{ .Values.jvmArgs.operatorLog }}
```

Comment:
shouldn't this be `.Values.jvmArgs.operatorLogConfig` or `.Values.jvmArgs.logConfig`

Reply:
@morhidi renamed to `logConfig`, assuming it should be shared.